client/Dockerfile

```diff
@@ -22,6 +22,10 @@ WORKDIR /app
 # Make composer super fast
 RUN composer global require hirak/prestissimo --no-plugins --no-scripts
 
+RUN apk update
+
+RUN apk add libpng libpng-dev libjpeg-turbo-dev libwebp-dev zlib-dev libxpm-dev gd && docker-php-ext-install gd
```

Comment:
Was this added for the CSV stuff? If so I think Symfony handles this natively with its Serializer - https://symfony.com/blog/new-in-symfony-3-2-csv-and-yaml-encoders-for-serializer

Reply:
As response below :)